### PR TITLE
fix: improve item price lookup for clue and loot

### PR DESCRIPTION
**Pull request by iProdigy**
Resolves an issue where coins could be thought to have 0 value instead of 1gp each (which @Supinic recently experienced when the clue notifier skipped a notification for [these items](https://i.imgur.com/0fzeOVz.png), given a value threshold of 20k)

### CODE REVIEW

File: src/main/java/dinkplugin/util/ItemUtils.java [hunk 1]
```diff
@@ -64,9 +59,17 @@ public boolean isItemNeverKeptOnDeath(int itemId) {
...
     }
 
-    public long getPrice(ItemManager itemManager, int itemId) {
-        int price = itemManager.getItemPrice(itemId);
-        return price > 0 ? price : itemManager.getItemComposition(itemId).getPrice();
+    public long getPrice(@NotNull ItemManager itemManager, int itemId) {
+        return getPrice(itemManager, itemId, null);
+    }
+
+    private int getPrice(@NotNull ItemManager manager, int itemId, @Nullable ItemComposition item) {
```
Comment (pajlada) [inline src/main/java/dinkplugin/util/ItemUtils.java:66]: Let's be consistent with the naming of the `ItemManager` variable as `itemManager` here

File: src/main/java/dinkplugin/util/ItemUtils.java [hunk 2]
```diff
@@ -27,12 +28,6 @@
 import java.util.stream.Stream;
 
 import static net.runelite.api.ItemID.*;
-import static net.runelite.api.ItemID.TWISTED_BRONZE_TROPHY;
-import static net.runelite.api.ItemID.TWISTED_DRAGON_TROPHY;
-import static net.runelite.api.ItemID.TWISTED_IRON_TROPHY;
-import static net.runelite.api.ItemID.TWISTED_MITHRIL_TROPHY;
-import static net.runelite.api.ItemID.TWISTED_RUNE_TROPHY;
-import static net.runelite.api.ItemID.TWISTED_STEEL_TROPHY;
```
Comment (pajlada) [inline src/main/java/dinkplugin/util/ItemUtils.java:35]: These imports weren't used since we already had a `*` include

**Review by pajlada — CHANGES_REQUESTED** [review]
Just a single nitpick, looks good :+1:

File: src/main/java/dinkplugin/util/ItemUtils.java (hunk 2 above)
Comment (iProdigy) [reply src/main/java/dinkplugin/util/ItemUtils.java:35]: correct

File: src/main/java/dinkplugin/util/ItemUtils.java (hunk 1 above)
Comment (iProdigy) [reply src/main/java/dinkplugin/util/ItemUtils.java]: Clarified in https://github.com/pajlads/DinkPlugin/pull/131/commits/85c73193d5bc0717e140b3a0092b6a4333db59a4

**Review by pajlada — APPROVED** [review]

**Pull request merged**